components/Heading.js

```diff
@@ -1,23 +1,96 @@
-import React from "react";
 import styled from "styled-components";
+import Link from "next/link";
+import { useState } from "react";
+import { GrYoga } from "react-icons/gr";
+import { BiHomeHeart } from "react-icons/bi";
+import { SlEnvolopeLetter } from "react-icons/sl";
+import { FeatureButton } from "./FeatureButton";
 
-const MainHeading = styled.h1`
-  padding-top: 15px;
-  text-align: center;
-  border: dotted;
-  font-size: 40px;
-  margin: auto;
+const HeadingContainer = styled.div`
   position: fixed;
-  top: 10px;
-  left: 50%;
-  translate: -50%;
-  background-color: #dbf3e1;
-  width: 95%;
-  color: #dbb290;
-  border-radius: 15px;
-  letter-spacing: 5px;
+  display: flexbox;
+  justify-content: space-between;
+  top: 15px;
+  right: 0;
+  left: 15px;
+`;
+
+const HamburgerModel = styled.nav`
+  display: flex;
+  flex-direction: column;
+  width: 25px;
+  height: 20px;
+  cursor: pointer;
+  justify-content: flex-start;
+  margin-top: 10px;
+`;
+
+const Line = styled.p`
+  width: 98%;
+  height: 2px;
+  background-color: #000;
+  margin: 2px;
+`;
+
+const Antonyoga = styled.h1`
+  display: flex;
+  font-size: 35px;
+  color: #dac9b6;
+  letter-spacing: 8px;
+  padding: 15px;
+  margin-top: -20px;
+  padding-right: 10px;
+`;
+
+const DropdownMenu = styled.ul`
+  display: ${({ isOpen }) => (isOpen ? "block" : "none")};
+  margin-left: -50px;
+  position: absolute;
+  top: 30px;
+  z-index: +1;
+`;
+
+const MenuItem = styled.li`
+  list-style: none;
+  margin-top: 30px;
 `;
 
 export default function Heading() {
-  return <MainHeading>ANTONYOGA</MainHeading>;
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <>
+      <HeadingContainer>
+        <HamburgerModel onClick={toggleMenu}>
+          <Line />
+          <Line />
+          <Line />
+        </HamburgerModel>
```

Comment:
For A11y reasons you should use onClick events only on buttons. Other elements with onclick events may not work properly for users with assistive technologies, like screen readers. Buttons also provide keyboard interactivity by default.